builds/e2e/templates/nested-get-secrets.yaml

```diff
@@ -4,10 +4,22 @@ steps:
     inputs:
       azureSubscription: $(az.subscription)
       KeyVaultName: $(azure.keyVault)
-      SecretsFilter: 'edgebuilds-azurecr-io-username,edgebuilds-azurecr-io-pwd,kvLogAnalyticWorkspaceId,kvLogAnalyticSharedKey,EdgeConnectivityStorageAccountConnString,EdgeLonghaulStorageAccountConnString,GitHubAccessToken'
+      SecretsFilter: >-
+        edgebuilds-azurecr-io-username,
+        edgebuilds-azurecr-io-pwd,
+        kvLogAnalyticWorkspaceId,
+        kvLogAnalyticSharedKey,
+        EdgeConnectivityStorageAccountConnString,
+        GitHubAccessToken,
+        edgebuild-blob-core-connection-string,
+        edgebuild-service-principal-secret,
+        edgebuild-1-PAT
+
   - task: AzureKeyVault@1
-    displayName: 'Nested Edge Azure Key Vault'
+    displayName: 'E2E Azure Key Vault'
     inputs:
       azureSubscription: $(az.subscription)
       KeyVaultName: $(azure.keyVault)
```

Comment:
Both these tasks use the same kv, $(azure.keyVault)? Should we just combine them entirely?

Reply:
That's the unfortunately reality. I wanted to separate secrets into 2 KV : E2E & the common KV. 
Given the development of how Nested LH & E2E pipeline tangle, it isn't going to be easy to separate the two.
I'm going to group the secrets as you suggested.